vllm/worker/habana_model_runner.py

```diff
@@ -425,6 +421,7 @@ class ModelInputForHPU(ModelRunnerInputBase):
     real_batch_size: Optional[int] = None
     batch_size_padded: Optional[int] = None
     virtual_engine: int = 0
+    lora_ids: Optional[List[int]] = None
     lora_mask: Optional[torch.Tensor] = None
```

Comment:
lora_mask & lora_logits_mask can be deleted here, right?

Reply:
These are deleted